pkg/component/gardenerapiserver/gardener_apiserver_test.go

```diff
@@ -643,6 +648,78 @@ var _ = Describe("GardenerAPIServer", func() {
 				}},
 			}},
 		}
+		clusterRole = &rbacv1.ClusterRole{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "gardener.cloud:system:apiserver",
+				Labels: map[string]string{
+					"app":  "gardener",
+					"role": "apiserver",
+				},
+			},
+			Rules: []rbacv1.PolicyRule{{
+				APIGroups: []string{"*"},
+				Resources: []string{"*"},
+				Verbs:     []string{"*"},
+			}},
+		}
+		clusterRoleBinding = &rbacv1.ClusterRoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "gardener.cloud:system:apiserver",
+				Labels: map[string]string{
+					"app":  "gardener",
+					"role": "apiserver",
+				},
+			},
+			RoleRef: rbacv1.RoleRef{
+				APIGroup: "rbac.authorization.k8s.io",
+				Kind:     "ClusterRole",
+				Name:     "gardener.cloud:system:apiserver",
+			},
+			Subjects: []rbacv1.Subject{{
```

Comment:
Why is [this](https://github.com/gardener/gardener/blob/06f3b19981f6ee4fdfb2471b87a1f2cb5bb9f84e/charts/gardener/controlplane/charts/application/templates/clusterrolebinding-apiserver.yaml#L17-L20) not required anymore?

Reply:
We don't need this use-case in `gardener-operator` - we will simply leverage the tokens provisioned by `gardener-resource-manager` in order to let GAPI talk to KAPI (similar to how we do it for other components like KCM).